docs/hub/_toctree.yml

```diff
@@ -214,6 +214,8 @@
         title: Livebook on Spaces
       - local: spaces-sdks-docker-shiny
         title: Shiny on Spaces
+      - local: spaces-sdks-docker-quarto
+        title: Quarto on Spaces
```

Comment:
Please move this after Giskard as this will display at the end of the current templates